openerp-atomfeed-service/src/main/resources/jobs-context.xml

```diff
@@ -200,8 +245,12 @@
                     <ref bean="openelisSaleOrderFeedFailedJobTrigger" />
                     <ref bean="eventRecordsNumberOffsetMarkerTrigger"/>
                     <ref bean="eventPublisherTrigger"/>
-                    <ref bean="openerpLabFeedJobTrigger" />
-                    <ref bean="openerpLabFeedFailedJobTrigger" />
+                    <ref bean="openerpLabTestFeedJobTrigger" />
```

Comment:
Please sequence the line as NormalEventA,FailedEventA, NormalEventB, FailedEventB, etc.

Reply:
Sure, will do that.